assets/src/blocks/Covers/CoversCarouselControls.js

```diff
@@ -0,0 +1,46 @@
+const { __ } = wp.i18n;
+
+const CoversCarouselArrow = ({ handler, direction, disabled = false }) => (
+  <button className={`carousel-control-${direction}`} disabled={disabled} onClick={handler}>
+    <span className={`carousel-control-${direction}-icon`} aria-hidden="true"><i></i></span>
+    <span className="visually-hidden">{__(direction === 'next' ? 'Next' : 'Prev', 'planet4-blocks')}</span>
+  </button>
+);
+
+export const CoversCarouselControls = ({
+  totalAmountOfCovers,
+  amountOfCoversPerRow,
+  slideCovers,
+  currentRow,
+}) => {
+  if (!amountOfCoversPerRow || totalAmountOfCovers <= amountOfCoversPerRow) {
+    return null;
+  }
+
+  const amountOfSlides = Math.ceil(totalAmountOfCovers / amountOfCoversPerRow);
+  const indicators = Array.from(Array(amountOfSlides).keys());
+
+  return (
+    <>
+      <CoversCarouselArrow
+        direction='prev'
+        disabled={currentRow === 1}
+        handler={() => slideCovers('prev')}
```

Comment:
Have you tried something like this?
```
<CoversCarouselArrow
        direction='prev'
        disabled={currentRow === 1}
        {...(slideCovers) && {handler: () => slideCovers('prev')}}
```
And then avoid to pass something like this:
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/734/files#diff-5e8e8bb129161a155bd56c85e46aea8d5d068edca136765f068b35f9286295c3R150

I haven't checked but I think it would work. What do you think?

Reply:
That's a good point! I've made the changes 👌